monodoc/monodoc.csproj

```diff
@@ -1,713 +1,27 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Had to switch to SDK style to get netstandard 2.0 support and the MSBuild Pack target.

Reply:
Also allows MASSIVELY simplifying the project file